internal/factsengine/json_schema/trento.checks.v1.FactsGathered.schema.json

```diff
@@ -0,0 +1,45 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
```

Comment:
This is basically the same one that we have in [wanda](https://github.com/trento-project/wanda/blob/main/priv/json_schema/trento.checks.v1.FactsGathered.schema.json), right?
And these two will converge in the contracts repo.